pkg/registrar/map_functions_test.go

```diff
@@ -471,6 +506,26 @@ var _ = Describe("MapFunctions", func() {
 
 						Expect(result).To(Equal(expected))
 					})
+
+					Context("workload has better matching supply chain", func() {
+						BeforeEach(func() {
+							workload.Labels["myOtherLabel"] = "myOtherLabelsValue"
+
+							sc := &v1alpha1.ClusterSupplyChain{
+								Spec: v1alpha1.SupplyChainSpec{
+									Selector: map[string]string{
+										"myLabel":      "myLabelsValue",
+										"myOtherLabel": "myOtherLabelsValue",
+									},
+								},
+							}
+
+							clientObjects = []client.Object{workload, sc}
```

Comment:
Pretty sure this should be
```
clientObjects = []client.Object{workload, sc, clusterSupplyChain}
```

removing `sc` should make the test fail, but in the current setup it does not.

Reply:
Good catch thanks!